### PR TITLE
Replaces the EB status box with cards

### PR DESCRIPTION
**Pull request by darrylmasson**
Also reindents some stuff, which is why the line change count is so high.

### CODE REVIEW

File: public/javascripts/status_script.js [hunk 1]
```diff
@@ -174,69 +173,72 @@ function UpdateDispatcher() {
...
-    $.getJSON("status/get_bootstrax_status", function(data) {
-      if (data.length == 0)
+  var timeout = 20*1000; // seconds since last update
+  for (var i in eventbuilders) {
+    $.getJSON("status/get_bootstrax_status?host="+eventbuilders[i], (data) => {
+      var h = "#"+eventbuilders[i]+"_";
+      if (typeof data.host == 'undefined') {
+        $(h+"status").html("offline");
+        $(h+"status").css("color", "gray");
+        $(h+"target").html("");
```
Comment (jorana) [inline public/javascripts/status_script.js:183]: I recently change this to `targets`. Also, can add the `post_processing` argument?

File: views/status.pug [hunk 2]
```diff
@@ -75,6 +75,17 @@ block content
       div.det_card
         strong Dispatcher is &nbsp;
         span(id="dispatcher_status")
+    each eb in eventbuilders
+      div.col-xs-6.col-sm-6.col-md-3(style='padding-left:0;padding-right:0')
+        div.det_card
+          strong #{eb} is &nbsp;
+          span(id=`${eb}_status`)
+          p
+            div(id=`${eb}_target`)
```
Comment (jorana) [inline views/status.pug:84]: also here I think?

**Review by jorana — COMMENTED** [review]
Thanks Darryl! I just noticed that some of the bootstrax I made to bootstrax aren't properly updated to this collection on the bootstrax side, let me double check.

**Comment by darrylmasson** [comment]
This branch hasn't been touched in six weeks, and I'm not currently working on it, but I'll keep it in mind if/when I get back to it

**Comment by darrylmasson** [comment]
This branch is sufficiently stale that it's not worth trying to revive it

**Pull request closed without merging**